Release cards 2 11 (#11289)
mlflow/mlflow: docs/source/new-features/index.rst

                    <div class="body">
                        <p>The <a href="../llms/transformers/index.html">transformers flavor</a> 
                        in MLflow has gotten a significant feature overhaul.</p>
                        <ul>
                            <li>All supported pipeline types can now be logged without restriction</li>
                            <li>Pipelines using foundation models can now be logged without copying the large model weights</li>
                        </ul>
                    </div>
                    <div class="tag">
                        <a href="https://github.com/mlflow/mlflow/releases/tag/v2.11.0">released in 2.11.0</a>
                    </div>
                </div>
            </div>
            <div class="grid-card">
                <div class="content-container">
                    <div class="header">
                        PEFT (Parameter-Efficient Fine-Tuning) support
                    </div>
                    <img class="card-image" src="../_static/images/logos/huggingface-logo.svg" alt="HuggingFace Logo"></img>
                    <div class="body">

>                         MLflow now natively supports PEFT (Parameter-Efficient Fine-Tuning) models in the Transformers flavor. PEFT 

Reviewer: ```suggestion
                        MLflow now natively supports `PEFT (Parameter-Efficient Fine-Tuning) <../llms/transformers/guide/index.html#peft-models-in-mlflow-transformers-flavor>`_ models in the Transformers flavor. PEFT 
```

self-update: same as the below comment

>                         unlocks significantly efficient model fine-tuning like LoRA, QLoRA, and Prompt Tuning. Check it out to 

Reviewer: ```suggestion
                        unlocks significantly efficient model fine-tuning like LoRA, QLoRA, and Prompt Tuning. Check out `the new QLoRA fine-tuning tutorial <../llms/transformers/tutorials/fine-tuning/transformers-peft.html>`_ to 
```

Self-update: "Learn more" doesn't stand out so adding in-line link.

                    </div>
                </div>
            </div>
            <div class="grid-card">
                <div class="content-container">
                    <div class="header">
                        Overhaul of MLflow Tracking UI for Deep Learning workflows
                    </div>
                    <img class="card-image" src="../_static/images/logos/mlflow-logo.svg" alt="MLflow"></img>
                    <div class="body">
                        We've listened to your feedback and have put in a huge amount of new UI features designed to empower and 
                        simplify the process of evaluating DL model training runs. Be sure to upgrade your tracking server and 
                        benefit from all of the new UI enhancements today!
                    </div>
                    <div class="tag">
                        <a href="https://github.com/mlflow/mlflow/releases/tag/v2.11.0">released in 2.11.0</a>
                    </div>
                </div>
            </div>
            <div class="grid-card">
                <div class="content-container">
                    <div class="header">

>                         Automated model checkpointing support added for Deep Learning model training

Reviewer: ```suggestion
                        Automated model checkpointing for Deep Learning model training
```

super-nit: To make the title renders within 2 rows in default and max page width (all other cards do)

Author: not a nit at all. Great idea :D

                    </div>
                    <img class="card-image" src="../_static/images/logos/mlflow-logo.svg" alt="MLflow"></img>
                    <div class="body">
                        When performing training of Deep Learning models with PyTorch Lightning or Keras, model checkpoint saving 

>                         is enabled, allowing for state storage during long-running training events and the ability to resume if 

Reviewer: nit: Do we have a doc reference for how to enabling and configuring the checkpointing?

Author: Good thing you pointed this out - we were missing the docstrings for tensorflow! Added those and updated with links to the API docs

                    <div class="header">
                        Keras 3 is now supported in MLflow
                    </div>
                    <img class="card-image" src="../_static/images/logos/keras-logo.svg" alt="Keras"></img>
                    <div class="body">
                        You can now log and deploy models in the new <a href="https://keras.io/keras_3/">Keras 3 format</a>, allowing you 
                        to work with TensorFlow, Torch, or JAX models with a new high-level, easy-to-use suite of APIs.
                    </div>
                    <div class="tag">
                        <a href="https://github.com/mlflow/mlflow/releases/tag/v2.11.0">released in 2.11.0</a>
                    </div>
                </div>
            </div>
            <div class="grid-card">
                <div class="content-container">
                    <div class="header">
                        MLflow now has support for OpenAI SDK 1.x
                    </div>
                    <img class="card-image" src="../_static/images/logos/openai-logo.svg" alt="OpenAI" style="max-height: 5rem"></img>
                    <div class="body">

>                         We've updated flavors that interact with the OpenAI SDK, bringing full support for the API changes with the 1.x release.

Reviewer: Do we have any good reference for the required code update for users? I guess it's only `task=openai.ChatCompletion` -> `task=openai.chat.Completion` and we've updated code examples in our doc ([here](https://github.com/mlflow/mlflow/pull/11123/files#diff-27092883ef9a4a81ab9b5ea59442ab48dd314babd61428270948fcfae0a80f28)). Maybe we can add a link to one of them as quick reference?
